langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModelListenerIT.java

```diff
@@ -46,8 +59,130 @@ protected Class<? extends Exception> expectedExceptionClass() {
         return ClientAuthenticationException.class;
     }
 
-    @Override
-    @Disabled("AzureOpenAiStreamingChatModel implementation is incorrect")
-    protected void should_listen_error() {
+    @Test
+    public void should_listen_request_and_response() {
```

Comment:
No need to duplicate tests from `StreamingChatModelListenerIT`, you could just remove the method

Reply:
Once this is done, we can merge!